Resources/public/js/views/universaldiscovery/ez-universaldiscoverybrowseview.js

```diff
@@ -67,14 +67,18 @@ YUI.add('ez-universaldiscoverybrowseview', function (Y) {
         },
 
         /**
-         * `multipleChange` event handler. It sets the selected view
-         * `addConfirmButton` flag according to the new `multiple` attribute value.
+         * `multipleChange` and `isSelectableChange` events handler. It sets the selected view
+         * `addConfirmButton` flag according to the new `multiple` attribute value and passes
+         * new `isSelectable` function to the selected view.
          *
-         * @method _setSelectedViewConfirmButton
+         * @method _setSelectedViewAttrs
          * @protected
          */
-        _setSelectedViewConfirmButton: function () {
-            this.get('selectedView').set('addConfirmButton', this.get('multiple'));
+        _setSelectedViewAttrs: function () {
+            this.get('selectedView').setAttrs({
```

Comment:
It's a bit weird to set 2 attributes when one changes, ie you should only set `addConfirmButton` when `multiple` changes and only set `isSelectable` when `isSelectable`changes.